contents/teams/experiments/objectives.mdx

```diff
@@ -1,29 +1,35 @@
-Motivation: The old query engine was difficult to extend and in some cases gave inconsistent results.
+#### Objective 1: Build advanced features  
+Motivation: Continue closing the gap with more mature competitors.
 
 What we will ship:
-* A more robust and accurate query engine
-* A new calculator to estimate running time
-* Enhanced visibility of experiment exposures and other health metrics
+* **Results timeseries** – Show the historical progression of experiment results to give a clearer picture over time, not just a snapshot. <TeamMember name="Juraj Majerik" photo />
+* **Experiment layers** – Allow users to be split between different experiments to avoid overlap. <TeamMember name="Rodrigo Iloro" photo />
+* **Multi-armed bandits** – Automatically adjust rollout based on live data. <TeamMember name="Anders Asheim Hennum" photo />
+
+#### Objective 2: Metrics + UX improvements  
+Motivation: Make experiments easier to set up, monitor, and understand. Reduce friction to improve retention.
 
-#### Objective 2: Add frequentist method + other requested features
 What we will ship:
-- **Frequentist Analysis**: Offer a frequentist alternative to our existing Bayesian approach, giving users more flexibility in how they interpret and report on experiment data.
-- **Data Warehouse Integration**: Finalize data warehouse support, so users can analyze external data within the experimentation workflow.
-- **Winsorization**: Implement optional outlier trimming (winsorizing) to handle extreme data points.
-- **Timeseries Deltas + Intervals**: Provide timeseries visualization of deltas and credible (or confidence) intervals for clearer trend insights over time.
-- **Breakdowns**: Allow users to break down experiment results on dimensions (i.e country, subscription plan, etc.)
+* **Setup wizard** – a simpler way to set up experiments, possibly with MaxAI assistance <TeamMember name="Rodrigo Iloro" photo />
+* **Activity log** – Show who made changes to an experiment and when <TeamMember name="Rodrigo Iloro" photo />
+* **Ratio metrics** – Enable metrics like "% of positive feedback" <TeamMember name="Anders Asheim Hennum" photo />
+* **Exclusion step** for funnel experiments <TeamMember name="Juraj Majerik" photo />
 
-#### Objective 3: Implement a fair and transparent pricing model
-Motivation: Monetize the experimentation product in a way that is straightforward for customers.
+#### Objective 3: Awesome documentation <TeamMember name="Juraj Majerik" photo />
+Motivation: Present ourselves as a mature and sophisticated product through excellent documentation.
 
 What we will ship:
-* A billing model that aligns cost with actual experiment usage and business value.
+* A clear, intuitive structure that supports both statistical methods and both versions of the query engine  
+* Detailed method explanations with examples  
+* Commonly used scenario walkthroughs  
+* AI-related content – Examples of how to run AI/prompt experiments
```

Comment:
```suggestion
* Easy ways to get to the relevant docs from within the app
```

eg the "read the docs" tooltips. Docs are only as good as they are easy to find and get to. The null hypothesis is that people won't go looking for docs, so won't read them.